Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec puma -C config/puma.rb
```

Comment:
Why do you need to use `Puma`?

Reply:
In rails documentation, recommend use `puma` as webserver, because with `puma` we can get maximum performance. [Puma Web Server](https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#adding-puma-to-your-application)

Also, we can use the next line:
`web: rails server`

Reply:
Puma covers some things like:
- Web concurrency, with the workers number
- Threads, to serve serve each request in a thread
- Port where app will listen.
- Timeout for request without answer.